modules/ROOT/partials/module-loading/webpack-dev-dependencies.adoc

```diff
@@ -41,3 +41,20 @@ npm install --save-dev \
 ----
 
 endif::[]
+ifeval::[{is_zip_install} == vite]
+. Create scaffolding for your project.
++
+[source,sh]
+----
+npm create vite@5 app && cd app && npm install tinymce && npm install --save-dev vite
```

Comment:
The `npm install --save-dev vite` at the end isn't needed here. `npm create vite` would already add vite as a dev dep

Reply:
what is `app` command after `npm create vite@5`? Just make sure it exists because the following commands will not be executed otherwise

Reply:
Yeah that's just the name of the app, it could be changed to the current dir. Maybe it should be that instead?
`npm create vite@5 . && npm i tinymce`
That will prompt the user for `y/n` if the CWD is not empty though

Reply:
updated using `npm create vite@5 . && npm install tinymce`